### PR TITLE
Models and migrations

### PR DESCRIPTION
**Pull request by weepotty**
# Description

Creates models and migrations for soundstate
Screenshot 2023-03-21 at 11 26 30 AM

Fixes # (issue)
https://trello.com/c/QJJBCksQ

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
- [x] Screenshot A - rails console

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added proof(eg. screenshots) that prove my fix is effective or that my feature works

### CODE REVIEW

File: app/models/event.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Event < ApplicationRecord
+  belongs_to :user
+
+  validates :title, presence: true, length: { maximum: 25 }
+  validates %i[acousticness danceability energy instrumentalness liveness speechiness tempo valence popularity], presence: true
```
Comment (cheehwatang) [inline app/models/event.rb:5]: I think we cannot put the keys in an array for the validation.
We would need to do something like:
:acousticness, :danceability, :energy, :instrumentalness, :liveness, :speechiness, :tempo, :valence, :popularity, presence: true
image
image

Comment (cheehwatang) [reply app/models/event.rb:5]: Another way to validate presence true is with validates_presence_of

validates_presence_of %i[acousticness danceability energy instrumentalness liveness speechiness tempo valence popularity]

File: app/models/event.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Event < ApplicationRecord
+  belongs_to :user
```
Comment (cheehwatang) [inline app/models/event.rb:2]: need to add association

has_many :playlists

File: app/models/event.rb (hunk 1 above)
Comment (weepotty) [reply app/models/event.rb:5]: yep you're right! sorry meant to save this PR as a draft. need to test out the validations then i'll add some screen shots of rails console

Comment (cheehwatang) [reply app/models/event.rb]: Everything else seems good. Learned a lot from the enum for the time in Event.

**Review by cheehwatang — APPROVED** [review]
Everything looks good

**Pull request merged**